### PR TITLE
Refactor template `register_public_key_client.html` for bootstrap compatible and alignment.

### LINKED ISSUE #3455: Inconsistent table alignments and font sizes
Tables are aligned at 20px and 40px; body/paragraph font-sizes vary from 14-18px. 

register a device welcome ka lite original

-------------------------------------
Fixes:
-Change 2nd table margins left and right to 40px;
-Change font size of bullet lists to 14px
-Adjust bottom margin of 1st table header to 15px

register a device welcome ka lite mock1

-------------------------------------
Additional Fixes:
-make bullet list stand out by changing font style to italics
-make sub headers of 2nd table stand out by increasing font-size to 20px and changing color to green

register a device welcome ka lite mock2

### PR DESCRIPTION
**Pull request by arceduardvincent**
Hi @aronasorman and @Joymysoy This fixes #3455 - Inconsistent table alignments and font sizes

Branch: Develop
Milestone: 0.14.x

This fix includes the alignment issue and remove the table  layout.
I also introduce the bootstrap compatibility in this template `register_public_key_client.html`.
It is now using `Grid System` of `bootstrap` to  achieve the nested grid.

I attached the image below to demonstrate the changes 
screen shot 2015-04-02 at 5 50 42 pm

### CODE REVIEW

File: python-packages/securesync/templates/securesync/register_public_key_client.html [hunk 1]
```diff
@@ -99,53 +74,46 @@
...
+            <li><em>{% trans "Your data will be backed up online" %}</em></li>
+            <li><em>{% trans "You can login online and review your usage data" %}</em></li>
+            <li><em>{% trans "Our team can use this data to objectively assess the impact our work is having around the world!" %}</em></li>
         </ul>
     </div>
 
-    <div class="chunk-container">
-        <h2>{% trans "Ready to register? You have two options:" %}</h2>
-
-        <table>
```
Comment (rtibbles) [inline python-packages/securesync/templates/securesync/register_public_key_client.html]: :+1: :+1: :+1:

File: python-packages/securesync/templates/securesync/register_public_key_client.html [hunk 2]
```diff
@@ -99,53 +74,46 @@
 
 
     {% if unregistered %}
-    <div class="chunk-container">
-        <h2>{% trans "What is device registration?" %}</h2>
-        <p class="normal-size" style="padding: 0px 40px 0px 40px;">
+    <div class="row">
+        <h2 class="col-md-12">{% trans "What is device registration?" %}</h2>
+        <p class="col-md-10" style="margin-left:40px">
```
Comment (jtamiace) [inline python-packages/securesync/templates/securesync/register_public_key_client.html]: please do not insert inline css changes, instead, insert these changes into one of the static files or in the style block at the top of this html page

Comment (jtamiace) [inline python-packages/securesync/templates/securesync/register_public_key_client.html]: There are several instances of this on the page. Other than that, great job rewriting the table!! :)

Comment (rtibbles) [inline python-packages/securesync/templates/securesync/register_public_key_client.html]: Note: as a general rule avoid in-line CSS and Javascript, this includes
inside script tags on the page. If in doubt, put it in khan-lite.css with a
comment block above and below, stating what it is for. As these styles are
applied across the site, you should also make sure your class names and
selectors are specific.

Comment (rtibbles) [inline python-packages/securesync/templates/securesync/register_public_key_client.html]: Also, check out the bootstrap typography classes to see if any of them
serve your purposes. We have bootstrap, so let us use it where we can.

Comment (arceduardvincent) [inline python-packages/securesync/templates/securesync/register_public_key_client.html]: Hi @rtibbles and @jtamiace thanks for the review.

I removed the `inline css` and make sure that the class names and selectors are unique. 
I used a prefix ` reg-dev` for the selectors and class names for this html.

**Pull request merged**